base_layer/core/src/covenants/decoder.rs

```diff
@@ -87,6 +87,7 @@ pub(super) trait CovenantReadExt: io::Read {
 }
 
 impl<R: io::Read> CovenantReadExt for R {
+    /// Reads next byte code
```

Comment:
Maybe docs for these?
- `pub struct CovenantTokenDecoder`
- `pub enum CovenantDecodeError`
- `pub(super) trait CovenantReadExt`